HelpSource/Tutorials/QT/things_you_wanna_know.schelp

```diff
@@ -0,0 +1,61 @@
+title:: QT : Things you probably want to know
+summary:: Miscellaneous things you probably want to know about QT.
+categories:: Tutorials>QT
+
+section:: Rendering isn't always available
+
+SuperCollider provides some useful classes to automate processes. Some of them are associated to a link::Classes/Clock:: : mainly link::Classes/Routine:: and link::Classes/Task::.
+
+strong::QT rendering system is unavailable within the context of clock associated threads::. In other words, you cannot 'directly' modify a widget within a link::Classes/Routine:: or a link::Classes/Task::.
+
+Instead, you need to defer the modification by strong::encapsulating it in a deferred function::, that will be executed once SuperCollider resume its main context. Doing otherwise will crash the program :
```

Comment:
```suggestion
Instead, you need to defer the modification by strong::encapsulating it in a deferred function::, that will be executed once SuperCollider resume its main context, see link::Classes/Function#-defer:: Doing otherwise will crash the program :
```

Think it would be good to link to https://doc.sccode.org/Classes/Function.html#-defer , but I am unsure about the actual syntax in this case, please check, have written it from memory and haven't tested it in actually rendered docs.